scalagen-sbt/src/sbt-test/scalagen/simple/build.sbt

```diff
@@ -0,0 +1,79 @@
+import java.nio.file.Paths
+
+import sbt._
+import scala.meta._
+import scala.meta.contrib._
+
+import Generators._
+
+version := "0.1"
+
+scalaVersion := "2.12.4"
+
+generators := Set(
+  Main
+)
+
+TaskKey[Unit]("checkUntouchedHello") := {
```

Comment:
Just a note. I agree we should have scripted tests, but I recommend we try to keep them as small and few as possible. Scripted tests are terribly inconvenient to write/debug/maintain in my experience. They are also super slow.

Reply:
:+1: I'll try to combine as many as possible.